src/constants.ts

```diff
@@ -0,0 +1,21 @@
+import { nbsp } from "utils";
+
+export const decimationMethods = [
+  {
+    value: 'LOSSY',
+    label: 'Lossy',
+    description: nbsp`Faster, but not as accurate`
```

Comment:
For the JavaScript unaware, this is a [template literal](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) with a tag function. The tag function is just a method call, but the interpreter does some magic to split apart the templated string into a list of arguments.

On second thought, there wasn't much of a reason to do this over a normal function, but it was fun to learn about 😁

The reason these strings need to have non-breaking spaces is because the Grafana dropdown was wrapping them, making it look cluttered and hard to read.

Reply:
Are the `description` blurbs here what shows up in the dropdown that you had posted about before? Where did we land on getting Melissa to review these? I realize we're trying to get this in today, so this doesn't need to gate the PR.

Reply:
I talked to Melissa and she was the inspiration for those descriptions. I just added her to the PR as well.